praw/models/reddit/message.py

```diff
@@ -33,8 +33,6 @@ class Message(InboxableMixin, ReplyableMixin, FullnameMixin, RedditBase):
     ``id``                  The ID of the message.
     ``name``                The full ID of the message, prefixed with 't4'.
     ``subject``             The subject of the message.
-    ``subreddit``           If the message was sent from a subreddit,
```

Comment:
Was this incorrect, or is it no longer provided?

Reply:
It's moved to SubredditMessage